Classes/Domain/Dto/Exif.php

```diff
@@ -11,88 +11,259 @@
  * source code.
  */
 
-/*
- * Exif DataType
- * See: http://www.exif.org/Exif2-2.PDF
+/**
+ * EXIF DataType
+ * @see http://www.cipa.jp/std/documents/e/DC-008-Translation-2016-E.pdf
  */
 class Exif extends AbstractMetaDataDto
 {
+    /**
+     * EXIF MetaData constructor
+     *
+     * @param array $properties
+     */
+    public function __construct(array $properties)
```

Comment:
Why do you think defining the default properties in the constructor is better than just defining it in the array?